### ISSUE TITLE
SCK Pad

### ISSUE THREAD

**Issue by ogamespec** [msinger/gbreveng#3]
I did a few more polishing runs and got more detailed images of the internal logic of the SCK pad.

![SCK_Fused](https://github.com/user-attachments/assets/5edfbdfb-aec4-40a4-b410-142a09e744d7)

![SCK_lap2_Fused](https://github.com/user-attachments/assets/004ee006-a2aa-4718-b3cf-370629e00cc0)

**Comment by ogamespec** [comment]
SCK pad research:
https://github.com/emu-russia/dmgcpu/blob/main/wiki/soc/pads.md#sck

**Comment by msinger** [comment]
Did you open a DMG CPU A chip as well? Or what revisions do you refer to?

**Comment by ogamespec** [comment]
No, I'm using https://siliconpr0n.org/map/nintendo/dmg-cpu-a/
But I'm "putting up" the polish from Revision B to find the differences between the revisions at the same time :-)

**Comment by ogamespec** [comment]
https://github.com/ogamespec/dmgcpu/blob/main/wiki/soc/pads.md#iobuf_c-pad_bidir_sck

I've added a little description for this pad

**Comment by msinger** [comment]
Yes, thanks. I've seen it. Makes sense to have a latch at the clock input.
